buildtest/cli/__init__.py

```diff
@@ -1140,6 +1140,14 @@ def report_menu(subparsers, parent_parser):
         "--format",
         help="format field for printing purposes. For more details see --helpformat for list of available fields. Fields must be separated by comma (usage: --format <field1>,<field2>,...)",
     )
+
+    filter_group.add_argument(
+        "-d",
+        "--detailed",
+        action="store_true",
+        help="print detailed report table with some fields by running buildtest report --detailed",
```

Comment:
This help message is a bit too technical and redundant. The user already knows its `buildtest report --detailed` so it should not be in the help message. The `buildtest report` prints test results it can be either in table format which is the default or raw text via `--terse` format.

So its not technically correct to say `print detailed report table` because the output doesnt have to be in table. The `--detailed` will just preselect some format fields that are not selected by default.

The help message can be 

```python
    filter_group.add_argument(
        "-d",
        "--detailed",
        action="store_true",
        help="Print a detailed summary of the test results",
```

Reply:
Yes, it does.